libvirtd-desktop/README.md

```diff
@@ -7,13 +7,33 @@
   ```
   $ sudo systemd-sysusers /usr/lib/sysusers.d/libvirt-qemu.conf
   ```
+- Create the `libvirtdbus` user and group:
+  ```
+  $ sudo bash -c 'getent group libvirtdbus >/dev/null || groupadd -r libvirtdbus'
+  $ sudo bash -c 'getent passwd libvirtdbus >/dev/null || \
+      useradd -r -g libvirtdbus -d / -s /sbin/nologin \
+      -c "Libvirt D-Bus bridge" libvirtdbus'
+  ```
+- Create the `libvirt` group to allow password-less polkit access to libvirt deamons:
+  ```
+  $ sudo bash -c 'getent group libvirt >/dev/null || groupadd -r libvirt'
+  ```
+- Optional: Add your UID to the libvirt group
+  ```
+  $ sudo usermod -G -a libvirt $YOUR-UID-HERE
+  ```
+- Copy the default libvirt dbus config and fix the selinux label
+  ```
+  $ sudo cp -a /usr/share/dbus-1/system.d/org.libvirt.conf /etc/dbus-1/system.d/
```

Comment:
This should already be the correct location. Maybe we need to ask dbus broker to reload the config instead:

```
$ sudo systemctl reload dbus-broker.service
```